### ISSUE TITLE
Contacting or seeing the full profile of user after search

### ISSUE THREAD

**Issue by mildanor** [alessandrocapra/jammin-react#4]
![screen shot 2016-12-14 at 13 57 50](https://cloud.githubusercontent.com/assets/16984074/21181202/75099c12-c205-11e6-94f1-7f6fbea143b4.png)

There should be a possibility to select to see the full profile of a user - either another button or by clicking on his name/picture.

Unregistered users after clicking to see the full profile or to contact should be prompted to either sign in or register to the service.

**Comment by alessandrocapra** [comment]
Seeing the profile works, but not the contact (still to define) @mildanor

**Comment by stefano-tech-ai** [comment]
Can you provide more details about what to do with the contact button? An easy thing to do might be to have a mailto href

**Comment by mildanor** [comment]
When looking at another users full profile, one should also be able to see contact me button so I added it. However it shouldn't be visible when you are looking at your own profile which is the case at the moment.

![screen shot 2016-12-21 at 01 33 41](https://cloud.githubusercontent.com/assets/16984074/21372075/cf81b98c-c71d-11e6-961f-a27e4192956d.png)

Could we fix?

And we didn't fully agree what the contact me button is going to do I think, perhaps it could be some way of emailing the user?Or sending him a message through the site? What would be the easiest to implement?

We could also leave it non-functional and I can write about how we ran out of time to implement this part in the final document.
@alessandrocapra what do you think?

**Comment by stefano-tech-ai** [comment]
Easiest thing is to open an email dialog when clicked. I Will do tomorrow

**Comment by alessandrocapra** [comment]
Hey guys, sorry t catch up so late. Actually this feature is not included in the one we decided to develop (at least if I recall correctly).
I was thinking that we could show a modal (http://www.w3schools.com/bootstrap/bootstrap_modal.asp) as an informative message to the teacher, explaining what this button will do if the functionality is developed.

What do you think? We can use the same idea in the other parts of the application of course

**Comment by mildanor** [comment]
@alessandrocapra yeah we could do!

I think our idea was that people can either contact each other individually or form jammer groups and even invite others to them. But of course we never planned on actually implementing it.. Because not enough time !.. So showing a modal could be a good solution in a sense that we could then write on how this is going to be developed further . I don't mind. Whichever is easiest to implement and most feasible time wise :)

**Comment by alessandrocapra** [comment]
@stefanomunarini I think the modal is the faster way, so whenever the teacher stumbles upon an interactive element that doesn't lead anywhere he will read what the interaction looks like after clicking there no?

**Comment by stefano-tech-ai** [comment]
I would use mailto href tbh, easy and at least you provide a way of doing what you are claiming with the button label. But up to you guys, the project is yours so you gotta decide it

**Comment by alessandrocapra** [comment]
Thanks Ste, I will see whether to pull this in or not, because we wrote in the document that we will allow to communicate within our system (not implementing, just that it will be present in the full version). Right @mildanor ? I am still in for trying to solve it with modals ;)

**Comment by mildanor** [comment]
We did. Actually, we wrote that users will have their own private area on Jammin site where they will be able to share music notes within groups that they will be able to form through our site. 

Either way we will have to write that we didnt implement this and concentrated on search functionality for now due to time constraints.

So we could say we have mail for the initial launch of the service and we will integrate the group thing and private area thing later or we can implement the modals and say that we will work on developing their functionality in the future for making fit for group forming and band material sharing. 

It all depends. I can come up with something for each case when writing report so either is fine for me ;P ;)
